lib/components/search.jsx

```diff
@@ -0,0 +1,202 @@
+'use strict'
+
+const figures = require('figures')
+const {
+  h,
+  Color,
+  Component,
+  Fragment,
+  Text
+} = require('ink')
+const SelectInput = require('ink-select-input')
+const Spinner = require('ink-spinner')
+const TextInput = require('ink-text-input')
+const libnpm = require('libnpm')
+
+
+const MAX_RESULTS = 10
```

Comment:
I don't really care to make something like this configurable. Just pick good defaults imo

Reply:
In [a separate thread](https://github.com/npm/tink/pull/11#discussion_r231722498) I was talking about changing the yargs options to pass down all the options of [`libnpmsearch`](https://www.npmjs.com/package/libnpmsearch), so this will just change to the default for the `limit` option, but allow it to be overridden by the user.